foam_controlwrapper/examples/poiseuille_parallel.py

```diff
@@ -0,0 +1,62 @@
+"""Example to solve 2D poiseuille flow
+
+"""
+
+from simphony.core.cuba import CUBA
+from simphony.engine import openfoamII
```

Comment:
There also currently seems to be no testing for this openfoamII wrapper to ensure that the wrapper (and mesh that it provides) supports the simphony-api?